editor/plugins/visual_shader_editor_plugin.h

```diff
@@ -158,6 +158,7 @@ class VisualShaderEditor : public VBoxContainer {
 
 	void _show_preview_text();
 	void _update_preview();
+	String _get_description(int i);
```

Comment:
It's a private editor method so it doesn't matter much, but I guess `p_idx` would be a better parameter name?

Reply:
Ah, true will rename when I come back

Reply:
Fixed in https://github.com/godotengine/godot/pull/31647